recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,176 @@
+
+    def requirements(self):
+        if self.options.qt:
+            self.requires("qt/5.15.1@bincrafters/stable")
+            self.options["qt"].shared = True
+            if tools.os_info.is_linux:
+                self.options["qt"].qtx11extras = True
+
+    def _system_package_architecture(self):
+        if tools.os_info.with_apt:
+            if self.settings.arch == "x86":
+                return ':i386'
+            elif self.settings.arch == "x86_64":
+                return ':amd64'
+
+        if tools.os_info.with_yum:
+            if self.settings.arch == "x86":
+                return '.i686'
+            elif self.settings.arch == 'x86_64':
+                return '.x86_64'
+        return ""
+
+    def build_requirements(self):
+        pack_names = None
+        if not self.options.minimal and tools.os_info.is_linux:
+            if tools.os_info.with_apt:
+                pack_names = [
+                    "freeglut3-dev",
+                    "mesa-common-dev",
+                    "mesa-utils-extra",
+                    "libgl1-mesa-dev",
+                    "libglapi-mesa",
+                    "libsm-dev",
+                    "libx11-dev",
+                    "libxext-dev",
+                    "libxt-dev",
+                    "libglu1-mesa-dev"]
+
+        if pack_names:
+            installer = tools.SystemPackageTool()
+            for item in pack_names:
+                installer.install(item + self._system_package_architecture())
+
+    def config_options(self):
+        if self.settings.compiler == "Visual Studio":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["BUILD_TESTING"] = "OFF"
+        cmake.definitions["BUILD_EXAMPLES"] = "OFF"
+        cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
+
+        if self.options.minimal:
+            cmake.definitions["VTK_Group_StandAlone"] = "OFF"
+            cmake.definitions["VTK_Group_Rendering"] = "OFF"
+        if self.options.ioxml:
+            cmake.definitions["Module_vtkIOXML"] = "ON"
+        if self.options.ioexport:
+            cmake.definitions["Module_vtkIOExport"] = "ON"
+        if self.options.qt:
+            cmake.definitions["VTK_Group_Qt"] = "ON"
+            cmake.definitions["VTK_QT_VERSION"] = "5"
+            cmake.definitions["VTK_BUILD_QT_DESIGNER_PLUGIN"] = "OFF"
+        if self.options.mpi:
+            cmake.definitions["VTK_Group_MPI"] = "ON"
+            cmake.definitions["Module_vtkIOParallelXML"] = "ON"
+        if self.options.mpi_minimal:
+            cmake.definitions["Module_vtkIOParallelXML"] = "ON"
+            cmake.definitions["Module_vtkParallelMPI"] = "ON"
+
+        if self.settings.os == 'Macos':
+            self.env['DYLD_LIBRARY_PATH'] = os.path.join(self.build_folder, 'lib')
+            self.output.info("cmake build: %s" % self.build_folder)
```

Comment:
What problem are you trying to fix here?

Reply:
Committed by mistake.
How I can check if cci Mac passed successfully without those lines, since they are commented out?

Reply:
In other words, how I get to know which OS configurations passed successfully at cci?

Reply:
Done